README.md

```diff
@@ -17,9 +17,10 @@ Lucide is trying to expand the icon set as much as possible while staying faithf
 
 ### Why choose Lucide over Feather Icons
 
-- Lucide already expanded the icon set by 130+ in less than a year, so more icons to work with.
+- More icons to work with : Lucide has already expanded Feather by over 550 icons.
```

Comment:
Nice catch, although I'd rather go with something more vague, so that we don't have to update this over and over and over again. :)
```suggestion
- More icons to work with: Lucide already has many times the icons Feather does.
```

Reply:
maybe change "the icons" to "the amount of icons"

Reply:
Or maybe, simpler : Lucide already has hundreds of icons more than Feather.

Reply:
> Or maybe, simpler : Lucide already has hundreds of icons more than Feather does.

Nice one, I like that, updated the suggestion accordingly.

Reply:
Comitted.